### ISSUE TITLE
Elfinder error message caused confusing.

### ISSUE THREAD

**Issue by nguyenhoa93** [imjoy-team/imjoy-rpc#42]
When I need to wait a while for my large image to display in the `itk-viewer` plugin, I saw this error message:

![img](https://i.imgur.com/niT00jZ.png)

If I didn't know what `elFinder` is, I would probably think that the Timeout error might be related to my running plugin, which caused confusion.

I think that the message should indicate that this plugin is optional and if users want to install it, they can go to https://github.com/imjoy-team/imjoy-elfinder, like this (display duration about 30s):
![img](https://i.imgur.com/ZiLNnhO.png)

Also, in [jupyter/README.md](https://github.com/imjoy-team/imjoy-rpc/blob/master/jupyter/README.md), we show the user that, for example: "Optionally, you can also add [elFinder](https://github.com/imjoy-team/imjoy-elfinder) plugin for imjoy jupyter extension. If you have not installed elFinder, following the instruction in [this repository](https://github.com/imjoy-team/imjoy-elfinder) to install."

**Comment by oeway** [comment]
Hi thanks for reporting! That is indeed confusing if you don't have elfinder installed. 
The recommended way to start jupyter notebook is to do `pip install imjoy` and start with `imjoy --jupyter` command. Like that we will install imjoy-elfinder.

For the notebook running on Binder, I am assuming it has already been installed.

How about we just add some text to indicate that to the elfinder part to ask the user to run `pip install imjoy-elfinder`?

**Comment by nguyenhoa93** [comment]
Yes, I agree to ask users to run `pip install imjoy-elfinder` instead of an error message because I think actually it is not an error. However, it only works for those who use local Jupyter notebook.

For users who use the Remote Jupyter notebook server, they also need to install the `jupyter-server-proxy` package to help the `elfinder` load successfully. If not, the same issue occurs: `Failed to load resource: the server responded with a status of 404 (Not Found)` for `elfinder`.
